deployed-contracts/governance.md

```diff
@@ -84,31 +85,70 @@ Specific functions that can be invoked by the guardians
 * Normal Timelock: [`0xce10739590001705F7FF231611ba4A48B2820327`](https://testnet.bscscan.com/address/0xce10739590001705F7FF231611ba4A48B2820327)
 * FastTrack Timelock: [`0x3CFf21b7AF8390fE68799D58727d3b4C25a83cb6`](https://testnet.bscscan.com/address/0x3CFf21b7AF8390fE68799D58727d3b4C25a83cb6)
 * Critical Timelock: [`0x23B893a7C45a5Eb8c8C062b9F32d0D2e43eD286D`](https://testnet.bscscan.com/address/0x23B893a7C45a5Eb8c8C062b9F32d0D2e43eD286D)
+* Omnichain Proposal Sender: [`0xCfD34AEB46b1CB4779c945854d405E91D27A1899`](https://testnet.bscscan.com/address/0xCfD34AEB46b1CB4779c945854d405E91D27A1899)
+
 
 ## Ethereum mainnet
 
 * Access Control Manager: [`0x230058da2D23eb8836EC5DB7037ef7250c56E25E`](https://etherscan.io/address/0x230058da2D23eb8836EC5DB7037ef7250c56E25E)
 * Guardian: [`0x285960C5B22fD66A736C7136967A3eB15e93CC67`](https://etherscan.io/address/0x285960C5B22fD66A736C7136967A3eB15e93CC67)
+* Normal Timelock: [`0xd969E79406c35E80750aAae061D402Aab9325714`](https://etherscan.io/address/0xd969E79406c35E80750aAae061D402Aab9325714)
+* FastTrack Timelock: [`0x8764F50616B62a99A997876C2DEAaa04554C5B2E`](https://etherscan.io/address/0x8764F50616B62a99A997876C2DEAaa04554C5B2E)
+* Critical Timelock: [`0xeB9b85342c34F65af734C7bd4a149c86c472bC00`](https://etherscan.io/address/0xeB9b85342c34F65af734C7bd4a149c86c472bC00)
+* Omnichain Governance Executor: [`0xd70ffB56E4763078b8B814C0B48938F35D83bE0C`](https://etherscan.io/address/0xd70ffB56E4763078b8B814C0B48938F35D83bE0C)
+* Omnichain Executor Owner Proxy: [`0x87Ed3Fd3a25d157637b955991fb1B41B566916Ba`](https://etherscan.io/address/0x87Ed3Fd3a25d157637b955991fb1B41B566916Ba)
+* Omnichain Executor Owner Implementation: [`0xfCaFda9672a120a55E98E1fDd96AC0402c71b4bB`](https://etherscan.io/address/0xfCaFda9672a120a55E98E1fDd96AC0402c71b4bB)
```

Comment:
I would not add the implementation address to reduce the cost of updating the docs. Users can discover the current implementation via Etherscan

```suggestion
```